app/src/main/java/com/ece493/group5/adjustableaudio/models/EqualizerPreset.java

```diff
@@ -5,59 +5,64 @@
 public class EqualizerPreset
```

Comment:
Not required, but you can make this class extend AudioData which will provide the necessary getters and setters for all the settings. It might look cleaner that way since we remove some repeated code. However this is not required.